src/utils/sound_loader.py

```diff
@@ -1,11 +1,33 @@
 from src.conf import config
+import os
+import glob
+import soundfile as sf
+
 import math
 import numpy as np
 import scipy.signal as sig
 
 class SoundLoader:
     def __init__(self):
         # tak się dobiera do ustalonych parametrów (wszystkie są w configurator.py)
+        print(Configurator.parse_analysis()['sampling_frequency'])
+
+
+    def sound_load_file():
+
+        i = 0
+        sound_list = [{}]
```

Comment:
Tu nie trzeba dawać {} w środku, i potem nie trzeba też będzie robić del sound_list[0]

Reply:
hmmm, ale jak nie dałam tego {} to mi wtedy wywalało błąd, że definiuję sound_list jako float i później tam dictionary usiłuję wsadzić :(

Reply:
żartowałam
działa XD